### PR TITLE
moyo_nav_bar_created

### PR DESCRIPTION
**Pull request by roman-zherebetskyi**
- [DEMO LINK](https://roman-zherebetskyi.github.io/layout_moyo-header/)
- [TEST REPORT LINK](https://roman-zherebetskyi.github.io/layout_moyo-header/report/html_report/)

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -1,14 +1,60 @@
...
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#homepage" class="logo__link">
+        <img src="images/logo.png" alt="logo.png">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#1" class="link is-active">apple</a>
```
Comment (Andr1yK) [inline src/index.html:27]:
```suggestion
            <a href="#1" class="link is-active">
              apple
            </a>
```

File: src/style.css [hunk 2]
```diff
@@ -1 +1,85 @@
...
+}
+
+.logo__link {
+  display: flex;
+}
+
+img {
+  width: 40px;
+  height: 40px;
+}
```
Comment (Andr1yK) [inline src/style.css:30]: don't style tag

File: src/style.css [hunk 3]
```diff
@@ -1 +1,85 @@
...
+
+.nav__list {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  align-items: center;
+}
+
+.nav__item {
+  display: flex;
```
Comment (Andr1yK) [inline src/style.css:40]: u need this?

File: src/style.css [hunk 4]
```diff
@@ -1 +1,85 @@
...
+
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  height: 100%;
+  padding: 23px 0;
+
+  transition: color 0.3s ease;
+
+}
```
Comment (Andr1yK) [inline src/style.css:65]:
```suggestion
  transition: color 0.3s ease;
}
```

File: src/style.css [hunk 5]
```diff
@@ -1 +1,85 @@
...
+
+.link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
```
Comment (Andr1yK) [inline src/style.css:73]:
```suggestion
.link:hover,
.is-active {
  color: #00acdc;
}
```

File: src/style.css [hunk 6]
```diff
@@ -1 +1,85 @@
...
+
+.is-active::after {
+  background-color: #00acdc;
+  border: 2px solid #00acdc;
+  border-radius: 8px;
+  position: absolute;
+  width: 100%;
+  display: block;
+  top: 56px;
```
Comment (Andr1yK) [inline src/style.css:82]: use bottom: 0

**Review by Andr1yK — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 1 above)
Comment (roman-zherebetskyi) [reply src/index.html:27]: +

File: src/style.css (hunk 2 above)
Comment (roman-zherebetskyi) [reply src/style.css:30]: got it! fixed +

File: src/style.css (hunk 3 above)
Comment (roman-zherebetskyi) [reply src/style.css:40]: yes, yes I need this

File: src/style.css (hunk 4 above)
Comment (roman-zherebetskyi) [reply src/style.css:65]: +

File: src/style.css (hunk 5 above)
Comment (roman-zherebetskyi) [reply src/style.css:73]: +

File: src/style.css (hunk 6 above)
Comment (roman-zherebetskyi) [reply src/style.css:78]: +

File: src/index.html [hunk 7]
```diff
@@ -1,14 +1,65 @@
...
+    >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#homepage" class="logo__link">
+        <img
+          class="img_logo"
+          src="images/logo.png"
+          alt="logo.png">
```
Comment (Andr1yK) [inline src/index.html:25]:
```suggestion
        <img
          class="img_logo"
          src="images/logo.png"
          alt="logo.png"
        >
```

File: src/style.css [hunk 8]
```diff
@@ -1 +1,82 @@
...
```
Comment (Andr1yK) [inline src/style.css:78]: use bottom: 0

And delete hardcoding (top: 56px)

**Review by Andr1yK — APPROVED** [review]

**Pull request closed without merging**